x-pack/test/fleet_functional/apps/home/welcome.ts

```diff
@@ -14,8 +14,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const PageObjects = getPageObjects(['common', 'home']);
   const kibanaServer = getService('kibanaServer');
 
-  // FLAKY: https://github.com/elastic/kibana/issues/109017
-  describe.skip('Welcome interstitial', () => {
+  describe('Welcome interstitial', () => {
```

Comment:
Going to kick off a flaky test runner job for this one. It passed locally.

Reply:
Flaky test runner job with 100 runs here: https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/6644